source/UIAUtils.py

```diff
@@ -203,3 +205,11 @@ def getValue(self,ID,ignoreMixedValues=False):
 			raise UIAMixedAttributeError
 		return val
 
+def shouldUseUIAConsole():
+	"Determines whether to use UIA in the Windows Console."

```

Comment:
Please wrap this in tripplets

```suggestion
	"""Determines whether to use UIA in the Windows Console."""
```